### PR TITLE
Allow disabling of automatic error logging #102

### PR DESCRIPTION
**Pull request by johannessjoberg**

### CODE REVIEW

**Review by johanssonanton — APPROVED** [review]

**Review by hussfelt — APPROVED** [review]
LGTM

File: test/errorHandling.js [hunk 1]
```diff
@@ -336,6 +340,17 @@ describe('Error Handling Tests:', function() {
       expect(result).to.deep.equal({ multiValueHeaders: { 'content-type': ['application/json'] }, statusCode: 500, body: '{"error":"This is a test error message"}', isBase64Encoded: false })
     }) // end it
 
+    it('Should not log error if option logger.errorLogging is false', async function() {
```
Comment (eriolo) [inline test/errorHandling.js:348]: Perhaps add a test for `errorLogging: true`?

**Review by eriolo — APPROVED** [review]
LGTM 🌮

File: test/errorHandling.js (hunk 1 above)
Comment (johannessjoberg) [reply test/errorHandling.js:348]: Good idea!

**Comment by johannessjoberg** [comment]
I've created a PR towards main repository 🤞 🚀

**Review by jalooc — APPROVED** [review]
Looks good.

On slack I suggested considering https://github.com/ds300/patch-package to deal with 3rd party package patching. What it does, is it enables you to patch _any_ package in your `node_modules` and does it in a super-flawless and legit way (it's well described in the `README`).

So why is it useful?
When you find out your 3rd party package has a flaw, you normally fork it and submit a PR. Then you either wait for it to be approved or link `package.json` directly to your fork. The disadvantage of first option is that you can't use your fix until the community agrees on it; the second is that you loose updates from the upstream and put yourself in a danger of locking yourself in your fork for ever (I've seen it in pretty much every single project I've ever worked on 😆). `patch-package` overcomes both those disadvantages by letting you stick to a proper version in `package.json`, yet have your fixes applied right away. The catch: works well rather for smaller fixes; vast changes can get you into trouble (although imho still less trouble than  not using it at all).

**Comment by johannessjoberg** [comment]
Thanks @jalooc 👍  Do you think we should switch to `patch-package` in this case?

**Comment by jalooc** [comment]
I think it's a good idea any time we make a fork with changes that we want to use in our project right away, to eliminate those 2 disadvantages. We can at least try to see if it's a nice workflow for us (I loved it in my own project ;) )

**Comment by johannessjoberg** [comment]
Closing since it's merged in main repository.

**Pull request closed without merging**